system_tests/monitoring.py

```diff
@@ -24,6 +25,7 @@
 from system_test_utils import unique_resource_id
 
 retry_404 = RetryErrors(NotFound)
+retry_404_500 = RetryErrors(NotFound, InternalServerError)
```

Comment:
@tseaver doesn't the argument need to be a tuple? (Sorry on my phone so peeking at source is difficult)